src/coredump/stacktrace.c

```diff
@@ -399,15 +403,124 @@ static int parse_core(int fd, const char *executable, char **ret, JsonVariant **
 
         free(buf);
 
+        if (r == -EINVAL)
+                log_warning("Failed to generate stack trace: %s", dwfl_errmsg(dwfl_errno()));
+        else if (r < 0)
+                log_warning_errno(r, "Failed to generate stack trace: %m");
```

Comment:
so this function will now log in some cases but not in others, our coding style usually suggests to either log about all or a bout none.

(also maybe add `, ignoring:` to the log messages

Reply:
changed the other returns to log - but these are not being ignored, the error is returned